package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "google-cloud-node",
+  "version": "1.0.0",
+  "description": "Google Cloud client libraries for Node.js",
+  "main": "bin/generate-readme.js",
+  "bin": "bin/generate-readme.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "scripts": {
+    "fix": "standard --fix",
```

Comment:
Eugh. I see the appeal here, but it is inconsistent with literally every other module we have.  I'll let it pass if you use `semistandard` instead.